Docs/Features/Rule-Matching.md

```diff
@@ -1,42 +1,48 @@
+
+> [!NOTE]
+> URLs typically have the `https://` prefix which also needs to be taken care by the rules you have written.
+
+## What about rulesets?
 
-> Note that URLs typically have `https://` prefix which also need to be taken care by the rules you have written. 
+Each ruleset is a collection of rules. You can have as many rules as you want inside a ruleset. Each ruleset is associated to a browser and you can have multiple rulesets for same browser. Additionally, rulesets are prioritized as per the ordering. If the same rule exists in two different rulesets, the ruleset that's higher in the list is triggered first.
 
 ## From UI
 
-Adding new rule can be done from UI by clicking `+ Rules` button in the Hurl main window.
+Adding a new rule can be done from UI by selecting **Rules** in the Hurl main window.
 
-- Use `Create` option to create a new Ruleset
-  - Choose the Browser and the available Alternate Launches
-  - Add any number of Rules with any rule type by selecting a rule type and entering the rule for each.
-  - Save!
-- Use `Test` to the newly added rules
+- Select **Create** to create a new ruleset
+  - Choose the browser and the available Alternate Launches
+  - Add any number of rules with any rule type by selecting a rule type and entering the rule for each
+  - Save
+- Select **Test** to the newly added rules
   - Enter the URL you want to test the rules with.
-  - `Test against Existing Rules` will test URL with above added rules.
-  - `Test with Rule` option can test against the specific rule you entered here.
-- Note that, you can rearrange Rulesets ordering from the UI, but not the rules in the ruleset itself.
-
+  - **Test against existing rules** will test URL with above added rules.
+  - **Test with rule** will test against the specific rule you entered here.
+- Note that you can rearrange rulesets ordering from the UI, but not the rules in the ruleset itself.
 
 ## From UserSettings.json
 
-You can add new rules into `AutoRoutingRules` property, which itself is a list of Rules Objects. Each rule object should contain 2 properties 
-  - `Rules` : it can be a single rule or multiple rules. (rules are prioritiesed, so first rule that matches will trigger the opening of browser)
-  - `BrowserName`: The value should be a Browser name that matches one of the `Browsers` names. which will be opened once a rule from the `Rules` property matches. Use `_Hurl` value to open Hurl
-  - `Name` To name the ruleset. Only for visual purposes. does not have any functionality.
-  - `AltLaunchIndex` to select the Alt Launch for a browser. It's a index number.
+You can add new rules into `AutoRoutingRules` property, which itself is a list of Rules Objects. Each rule object should contain 2 properties:
+
+- `Rules`: It can be a single rule or multiple rules. Rules are prioritized: the first rule that matches will trigger the opening of browser.
```

Comment:
```suggestion
- `Rules`: It can be a single rule or multiple rules. Rules are prioritized: the first rule that matches will trigger the opening of a browser.
```